### PR TITLE
[AND-3314] Refactor vungle banner adapter. 

### PR DESCRIPTION
**Pull request by felixzhang0703**
https://vungle.atlassian.net/browse/AND-3314
VungleNativeAd is deprecated in 6.10.x version. We will use Banner to replace mrec.

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java [hunk 1]
```diff
@@ -254,26 +250,7 @@ private void createBanner() {
...
-
-        updateVisibility(mVisibility);
-        adView.setLayoutParams(adParams);
-        // don't add to parent here
-        listener.onAdAvailable();
-      } else {
-        // missing resources
-        listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
-      }
+      listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java:253]: minor: we can remove this block or propose to use error code `AdRequest.ERROR_CODE_INVALID_REQUEST`
refer to: https://github.com/Vungle/googleads-mobile-android-mediation/blob/ccfd2c6cdccc5a6409482c81a0626bcbb97cc3a3/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java#L247-L253

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java [hunk 2]
```diff
@@ -33,11 +32,6 @@
    */
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java:32]: We should also update this comment similar to: `Vungle ad object for banner ads.`

**Review by alexander-gubov — APPROVED** [review]

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java (hunk 1 above)
Comment (felixzhang0703) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java]: Use ERROR_CODE_INVALID_REQUEST.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java (hunk 2 above)
Comment (felixzhang0703) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java:32]: Resolved.

**Review by HuFangshuai — APPROVED** [review]
LGTM

**Comment by HuFangshuai** [comment]
@felixzhang0703 For reminder due to AdMob cla check, we can't choose `Create a merge commit` when merging PR. Feel free to use the other two options.

**Pull request merged**